app/controllers/containers_controller.rb

```diff
@@ -48,6 +48,27 @@ def commit
       { :container => @container, :e => e }
   end
 
+  def power
+    compute_resource = @container.compute_resource
+    @vm = compute_resource.find_vm_by_uuid(@container.uuid)
+    run_vm_action(@vm.ready? ? :stop : :start)
+  end
+
+  def run_vm_action(action)
+    if @vm.send(action)
```

Comment:
For some reason the stop action fails for me every time if I don't give a [timeout](https://github.com/swipely/docker-api/blob/master/lib/docker/container.rb#L173) , I can start them easily but not stop them, have you ran into this issue too?